shared/src/commonMain/kotlin/org/hyperskill/app/topics_repetitions/view/mapper/TopicsRepetitionsViewDataMapper.kt

```diff
@@ -55,10 +55,15 @@ class TopicsRepetitionsViewDataMapper(
             } else {
                 ShowMoreButtonState.EMPTY
             },
-            topicsToRepeatWillLoadedCount = min(
-                state.topicRepetitionStatistics.totalCount - state.topicsRepetitions.count(),
-                TopicsRepetitionsActionDispatcher.TOPICS_PAGINATION_SIZE
-            )
+            topicsToRepeatWillLoadedCount = if (state.topicsRepetitions.size % TopicsRepetitionsActionDispatcher.TOPICS_PAGINATION_SIZE != 0) {
```

Comment:
It's better to extract that condition into Feature static function. You used the same code in reducer.